addons/point_of_sale/static/tests/pos/tours/utils/product_screen_util.js

```diff
@@ -888,6 +888,17 @@ export function setTimeZone(testTimeZone) {
     };
 }
 
+export function setDateTime(testDateTime) {
```

Comment:
You already have `freezeDateTime` in `point_of_sale/static/tests/pos/tours/utils/chrome_util.js` which does basically the same :)